### ISSUE TITLE
TypeError: o.getFiberRoots is not a function

### ISSUE THREAD

**Issue by tpiersall-reifyhealth** [forge-42/react-router-devtools#138]
After installing and running the application I am getting this error when using Chrome. Has anyone else seen this?  I am actually getting this error when trying to just use the docs site of remix-dev-tools. I was able to open it in Safari though.

`TypeError: o.getFiberRoots is not a function`

**Comment by AlemTuzlak** [comment]
I never seen this happen, I might know why though, could you create a repro somehow?

**Comment by tpiersall-reifyhealth** [comment]
I can't send my exact one but I will try to spin up a new one and see if I can replicate it later today.  In the meantime, just to check, are you seeing this when you use chrome for the docs site?

<img width="1009" alt="image" src="https://github.com/forge42dev/Remix-Dev-Tools/assets/68873018/bb856aa6-2a33-4fcc-9063-9c8f975c6d1f">

**Comment by AlemTuzlak** [comment]
No, could it be that you have something disabled in the browser?

**Comment by tpiersall-reifyhealth** [comment]
Possibly.  It's a company machine so they might have some permissions and or something that it being managed.  I'll check it out.  Thanks!
